flight/Modules/Stabilization/stabilization.c

```diff
@@ -941,13 +978,22 @@ static void SettingsUpdatedCb(UAVObjEvent * ev)
 		// The dT has some jitter iteration to iteration that we don't want to
 		// make thie result unpredictable.  Still, it's nicer to specify the constant
 		// based on a time (in ms) rather than a fixed multiplier.  The error between
-		// update rates on OP (~300 Hz) and CC (~475 Hz) is negligible for this
-		// calculation
+		// update rates on OP (~300 Hz) and CC (~475 Hz) is negligible for this calculation
+		//fakeDt now only used for vbar_decay, for gyro filter now a real, but filtered, dT is used
 		const float fakeDt = 0.0025f;
-		if(settings.GyroTau < 0.0001f)
-			gyro_alpha = 0;   // not trusting this to resolve to 0
+
+		if (settings.GyroCutoff < 0.0001f) {	// not trusting this to resolve to 0; disable LPF when GyroCutoff = 0
```

Comment:
I thought the user has to decide what he wants.  
I also think low values are not very useful, as the delay gets very large, and we filter useful information I was not sure where a border for disabling shout be.

I implemented only 0 or very low cutoff frequencies disabled the LPF.
But when you want it to be at e.g. 5 Hz, i can to do it.